android/libraries/rib-router-navigator/src/main/kotlin/com/uber/rib/core/RouterNavigator.kt

```diff
@@ -118,18 +115,16 @@ interface RouterNavigator<StateT : RouterNavigatorState> {
    * Switch to a new state - this will switch out children if the state is not the current active
    * state already.
    *
-   *
-   * NOTE: This will retain the Riblet in memory until it is popped. To push transient, riblets,
-   * use [RouterNavigator.pushTransientState]
-   *
+   * NOTE: This will retain the Riblet in memory until it is popped. To push transient, riblets, use
```

Comment:
This is probably need some adjusting after your changes. We'll not retain Riblet in memory anymore. Should we deprecate the DEFAULT flag?

Reply:
I will modify comment for the cleaner understanding. "Memory" mentioned in the comment means navigation stack memory () not the physical heap memory. Changes in this PR doesn't affect any navigation logic, so `DEFAULT` flag  still will do the same things.